test-app/src/widgets/ClearButton.js

```diff
@@ -0,0 +1,41 @@
+import _, { noop } from 'lodash';
+import Button from './Button';
+
+export default class CrossButton extends Button {
+  constructor(options, tabIndex, callBack = _.noop) {
+    super(options, tabIndex, callBack);
+  }
+
+  _fillBackground(color) {
+    this.background.graphics.beginFill(color).drawCircle(0, 0, this.height * 0.5);
+  }
+
+  _onMouseDown() {
```

Comment:
The original behavior in Button seems to cause button click area issues, which should be fixed in a later PR, so I just replaced it with a noop for the ClearButton for now